recipes/hail/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "hail" %}
+{% set version = "0.2.34" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: 9c3a4e5f81ad23b569c1d6dc80bfb070
+  
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - python3 >=3.6,<3.7
+  host:
+    - bokeh <0.14
+    - decorator <5
+    - ipykernel <5
+    - matplotlib <3
+    - numpy <2
+    - pandas >0.22,<0.24
+    - parsimonious <0.9
+    - pip
+    - pyspark >=2.4,<2.4.2
+    - python3 >=3.6,<3.7
+    - requests >=2.21.0,<2.21.1
+    - seaborn <0.9
+  run:
+    - bokeh <0.14
+    - decorator <5
+    - ipykernel <5
+    - matplotlib <3
+    - numpy <2
+    - pandas >0.22,<0.24
+    - parsimonious <0.9
+    - pyspark >=2.4,<2.4.2
+    - python3 >=3.6,<3.7
+    - requests >=2.21.0,<2.21.1
+    - seaborn <0.9
+
+test:
+  imports:
+    - hail
+  commands:
+    - hail --help
+
+about:
+  home: https://hail.is/
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: "Scalable genetic data analysis"
```

Comment:
You need to add yourself as a maintainer:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L78-L83